File: TransCoder_tokenized_test_set_functions/java/CHECK_LINE_TOUCHES_INTERSECTS_CIRCLE.java
static void checkCollision ( int a, int b, int c, int x, int y, int radius ) {
  double dist = ( Math . abs ( a * x + b * y + c ) ) / Math . sqrt ( a * a + b * b );
  if ( radius == dist ) System . out . println ( "Touch" );
  else if ( radius > dist ) System . out . println ( "Intersect" );
  else System . out . println ( "Outside" );
}

File: TransCoder_tokenized_test_set_functions/python/CHECK_LINE_TOUCHES_INTERSECTS_CIRCLE.py
def checkCollision ( a, b, c, x, y, radius ) :
    dist = ( ( abs ( a * x + b * y + c ) ) / math . sqrt ( a * a + b * b ) )
    if ( radius == dist ) :
        print ( "Touch" )
    elif ( radius > dist ) :
        print ( "Intersect" )
    else :
        print ( "Outside" )
    
